search-sentiments-map/routes/sentiment.js

```diff
@@ -1,53 +1,59 @@
 /** Server-side script that uses the Google Cloud Natural Language API to get the sentiment score. */
 const express = require('express');
-var router = express.Router();  // Use a Router to divide the app into modules.
+var router = express.Router();  // Using Router to divide the app into modules.
+var allSettled = require('promise.allsettled');
 const language = require('@google-cloud/language');
 
-var bodyParser = require('body-parser');  
-// Create application/x-www-form-urlencoded parser.
-var urlencodedParser = bodyParser.urlencoded({ extended: false })
-
-/** Returns the sentiment score of inputted search results. */  
-router.post('/', urlencodedParser, (req, res) => { 
-  const searchTopic = req.body.searchTopic;
-  console.log('ntarn debug: '+ searchTopic);
-  getSentimentScore(searchTopic).then(score => {
-    // Prepare output in JSON format.  
-    response = {  
-      sentimentScore:score,  
-    };  
-    // TODO(ntarn): Remove console.log statement when finished with feature. 
-    console.log('ntarn debug: searchTopic score' + response.sentimentScore);  
-    res.end(JSON.stringify(response));
-  }).catch(err => {
-    console.log(err);
-  })
+var bodyParser = require('body-parser');
+var textParser = bodyParser.text();
+
+router.post('/', textParser, (req, res) => {
+  console.log('Got body:' + req.body);
+  getSentimentScore(req.body)
+    .then((sentimentScore) => {
```

Comment:
4 space indent.

Reply:
hmm, now I'm not sure can you send me documentation in a private message, and then change if it you're correct when addressing your PR commits?

Reply:
I actually just ended up moving line 12(the "then" line) onto line 11, it all fits on one line.